emukit/examples/profet/meta_benchmarks/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .meta_svm import meta_svm
```

Comment:
I don't think I fully understand the "meta" naming idea. Can you explain please?

Reply:
The 'meta' part comes from the fact that we are not actuallz optimize the hyperparameter of a SVM but instead rather something like an emulation if you want (generated by a meta-model) that mimics the hyperparameter optimization problem of a SVM. I will submit a tutorial notebook in an extra PR that explains the whole idea in more depth.